src/AppInstallerCommonCore/Public/winget/Experiment.h

```diff
@@ -41,23 +41,23 @@ namespace AppInstaller::Settings
 
         using Key_t = std::underlying_type_t<Key>;
 
-        Experiment(std::string_view name, std::string_view jsonName, std::string_view link, std::string key) :
-            m_name(name), m_jsonName(jsonName), m_link(link), m_key(key) {}
+        Experiment(std::string name, std::string jsonName, std::string link, std::string key) :
+            m_name(std::move(name)), m_jsonName(jsonName), m_link(std::move(link)), m_key(std::move((key))) {}

```

Comment:
```suggestion
            m_name(std::move(name)), m_jsonName(std::move(jsonName)), m_link(std::move(link)), m_key(std::move((key))) {}
```
Accompanied by making `m_jsonName` not a view.